.github/workflows/regression-tests.yml

```diff
@@ -136,6 +140,21 @@ jobs:
           path: core/test-results/
           retention-days: 30
 
+      - name: Send slack notification
+        uses: slackapi/slack-github-action@v1.26.0
+        if: failure()
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        with:
+          payload: |
+            {
+              "Job": "${{ github.job }}",
+              "Status": "Fail",
```

Comment:
Let's change this to `Failed` as it's a past occurrence when we receive the notification. (Same below)

Reply:
I changed this and probably set it back during rebase 😮‍💨